<launch>
  <node name="referee_node" pkg="uml_3d_race" type="referee" output="screen"/>

  <node name="mover_node" pkg="uml_3d_race" type="mover_node" output="screen"/>
</launch>
